### PR TITLE
Fix binding power bug

### PR DESCRIPTION
**Pull request by woodrowbarlow**

### CODE REVIEW

**Comment by woodrowbarlow** [comment]
@jdpage i'm not totally clear on why my change seemingly fixed the bug.

File: tests/test_ast.py [hunk 1]
```diff
@@ -377,7 +387,8 @@ def test_isr_def_empty():
 def test_isr_def():
```
Comment (woodrowbarlow) [inline tests/test_ast.py]: this is the test that was failing (after converting it to have multiple statements).
a similar test for functions would also fail.

the test i added above (`test_let_multistatement`) never failed.

**Comment by woodrowbarlow** [comment]
@jdpage i'm not totally clear on why my change seemingly fixed the bug.

**Comment by jdpage** [comment]
If it fixes the bug and all the tests pass, it's probably correct; feel free to merge. 

I'm not entirely clear on why it fixes it either, though. I'll have to sit down and trace through the code.

**Pull request merged**